tests/context.test.tsx

```diff
@@ -63,22 +64,32 @@ it('uses context store with selectors', async () => {
 })
 
 it('uses context store api', async () => {
-  const { Provider, useStoreApi } = createContext<CounterState>()
+  const { Provider, useStoreApi: useStoreApiOrig } =
+    createContext<CounterState>()
 
   const createStore = () =>
-    create<CounterState>((set) => ({
-      count: 0,
-      inc: () => set((state) => ({ count: state.count + 1 })),
-    }))
+    create(
+      subscribeWithSelector<{ count: number; inc: () => void }>((set) => ({
+        count: 0,
+        inc: () => set((state) => ({ count: state.count + 1 })),
+      }))
+    )
+
+  const useStoreApi = useStoreApiOrig as () => Omit<
+    StoreApi<CounterState>,
+    'subscribe'
+  > & {
+    subscribe: ReturnType<typeof createStore>['subscribe']
+  }
```

Comment:
I assume this is a temporary solution to use `subscribe` on context stores, right? Or is there another way?

Reply:
Good point. #606 for a better way.